packages/framework-provider-kubernetes-infrastructure/src/helpers/logger.ts

```diff
@@ -1,9 +1,21 @@
-import { Logger } from '@boostercloud/framework-types'
+import { Level, Logger } from '@boostercloud/framework-types'
 
-export const scopeLogger = (scope: string, logger: Logger): Logger => {
+export const scopeLogger = (scope: string, logger: Logger, level = Level.info): Logger => {
   const newLogger: Logger = { ...logger }
-  newLogger.debug = (...args: any[]) => logger.debug([`[${scope}]`, ...args].join(' '))
-  newLogger.info = (...args: any[]) => logger.info([`[${scope}]`, ...args].join(' '))
-  newLogger.error = (...args: any[]) => logger.error([`[${scope}]`, ...args].join(' '))
+  newLogger.debug = (...args: any[]) => {
+    if (level <= Level.debug) {
```

Comment:
Just for curiosity, if you are alway calling the specific logger method why you need the level?

Reply:
we added a lot of debug messages during the development of the K8s provider for debugging and here we are muting all those debug messages defaulting the logger to info level. 

This will be refactored after the meeting to use the application logging level

Reply:
This function shouldn't be doing this, we should have another function that initializes the logger this way. Let's do that: Remove these checks from here and add them to the initialization